perllib/FixMyStreet/Script/ArchiveOldEnquiries.pm

```diff
@@ -135,22 +138,27 @@ sub close_problems {
     return unless $opts->{commit};
 
     my $problems = shift;
+    my $extra = { auto_closed_by_script => 1 };
+    $extra->{is_superuser} = 1 if !$opts->{user_name};
```

Comment:
Does the `auto_closed_by_script` not belong with the report rather than the comment? I guess it doesn't make much difference as we can work back from the comments if we need to.